Yafc.Parser/Data/FactorioDataDeserializer_Entity.cs

```diff
@@ -557,7 +574,7 @@ void parseEffect(LuaTable effect) {
 
         // These types have already called ReadEnergySource/ReadFluidEnergySource (generator, burner generator) or don't consume energy from YAFC's point of view (pump to EII).
         // TODO: Work with AAI-I to support offshore pumps that consume energy.
-        if (factorioType is not "generator" and not "burner-generator" and not "offshore-pump" and not "solar-panel" and not "accumulator" and not "electric-energy-interface"
+        if (factorioType is not "generator" and not "burner-generator" and not "offshore-pump" and not "solar-panel" and not "accumulator" and not "electric-energy-interface" and not "lightning-attractor"
```

Comment:
I would suppose that a "not in 'known types' list" would work better here. Something like

```c#
if (not knownTypes.Contains(factorioType)
```

Then we can simply add a new type to the list (if we'll ever need a new type), without worrying whether this (and other places?) properly get updated or not?

And it adds readability, as this becomes quite a long if statement...